Description: define __aligned_u64 for use with new linux-libc-dev
 including linux/types.h will not help; klibc includes kernel sources
 without defining _KERNEL_, so certain types are not available
 .
 The person named in the Author field signed this changelog entry.
Author: Adam Conrad <adconrad@ubuntu.com>

---
--- klibc-1.5.25.orig/usr/include/net/if_arp.h
+++ klibc-1.5.25/usr/include/net/if_arp.h
@@ -1,3 +1,6 @@
 /* if_arp.h needs sockaddr */
 #include <sys/socket.h>
+#ifndef __aligned_u64
+#define __aligned_u64 __u64 __attribute__((aligned(8)))
+#endif
 #include <linux/if_arp.h>
--- klibc-1.5.25.orig/usr/include/net/if_packet.h
+++ klibc-1.5.25/usr/include/net/if_packet.h
@@ -1 +1,4 @@
+#ifndef __aligned_u64
+#define __aligned_u64 __u64 __attribute__((aligned(8)))
+#endif
 #include <linux/if_packet.h>
